internal/core/thirdparty/tantivy/tantivy-binding/src/tokenizer.rs

```diff
@@ -127,30 +157,34 @@ impl AnalyzerBuilder<'_>{
                     // build with filter if filter param exist
                     builder=self.build_filter(builder, value)?;
                 },
-                "max_token_length" => {
```

Comment:
why delete this?

Reply:
tokenizer max token length in es means split long token not remove long token, but now we don't have same function, so remove this option.